### DISCUSSION TITLE
USB Midi Host integration possible?

### DISCUSSION THREAD

**Discussion by heyudude** [GillesLACAUD/OPLA-Source-code#56] [Q&A]
Hi,

I came across this project and I was wondering if it was possible to integrate a USB Midi Host like this one.
The small PCB interacts with the ESP32 also: https://www.instructables.com/Arduino-USB-to-Legacy-MIDI-Converter/

**Comment by GillesLACAUD** [comment]
Hi Dude
I think it is possilbe to use one of the USB on the ESP32 Board to make this...

**Reply by heyudude** [reply]
My compliments for the setup and detailed instructions! Do you know when the kits are back on stock? Otherwise I need to order at Aliexpress or elsewhere.

**Reply by GillesLACAUD** [reply]
https://salmacis.wixsite.com/hansysynth/copy-of-ttgo

**Reply by GillesLACAUD** [reply]
it is here for the kit
